content/en/docs/instrumentation/python/_index.md

```diff
@@ -13,19 +13,79 @@ the generation and collection of application telemetry data such as metrics,
 logs, and traces. This documentation is designed to help you understand how to
 get started using OpenTelemetry for Python.
 
+## Version support
+
+OpenTelemetry-Python supports Python 3.6 and higher.
+
 ## Status and Releases
 
 The current status of the major functional components for OpenTelemetry Python
 is as follows:
 
-| Tracing   | Metrics | Logging      |
-| -------   | ------- | ------------ |
-| Stable    | Alpha   | Experimental |
+- **Tracing** ([API][api/t], [SDK][sdk/t]): [Stable][]
+- **Metrics** ([API][api/m], [SDK][sdk/m]): [Alpha][Experimental]
+- **[Logging][]**: [Experimental][]
+
+[api/m]: https://opentelemetry-python.readthedocs.io/en/stable/api/metrics.html
+[api/t]: https://opentelemetry-python.readthedocs.io/en/stable/api/trace.html
+[Experimental]: /docs/reference/specification/versioning-and-stability/#stable
+[Logging]: https://opentelemetry-python.readthedocs.io/en/stable/sdk/logs.html
+[sdk/m]: https://opentelemetry-python.readthedocs.io/en/stable/sdk/metrics.html
+[sdk/t]: https://opentelemetry-python.readthedocs.io/en/stable/sdk/trace.html
+[Stable]: /docs/reference/specification/versioning-and-stability/#stable
 
 {{% latest_release "python" /%}}
 
-## Further Reading
+## Installation
+
+The API and SDK packages are available on PyPI, and can installed via pip:
+
+```console
+$ pip install opentelemetry-api
+$ pip install opentelemetry-sdk
+```
+
+In addition, there are several extension packages which can be installed separately as:
+
+```console
+$ pip install opentelemetry-exporter-{exporter}
+$ pip install opentelemetry-instrumentation-{instrumentation}
+```
+
+These are for exporter and instrumentation packages respectively.
+The Jaeger, Zipkin, OTLP and OpenCensus Exporters can be found in the [exporter](https://github.com/open-telemetry/opentelemetry-python/blob/main/exporter/)
+directory of the repository. Instrumentations and additional exporters can be found in the
+[Contrib repo instrumentation](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation)
+and [Contrib repo exporter](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/exporter) directories.
+
+## Extensions
+
+To find related projects like exporters, instrumentation libraries, tracer
+implementations, etc., visit the [Registry](/registry/?s=python).
+
+### Installing Cutting Edge Packages
+
+While the project is pre-1.0, there may be significant functionality that
```

Comment:
This should probably be rephrased as the project is beyond 1.0 :D Some packages (many instrumentations for example) are still pre 1.0

Reply:
@codeboten - how about:

> There is some functionality that has not yet been released to PyPI.

Reply:
Changed. PTAL